configuration.md

```diff
@@ -13,6 +13,7 @@ Casual supports the following configuration options which can be set using envir
     See [Inbound Startup Configuration](inbound.md#startup-configuration) for more details.
 * `CASUAL_OUTBOUND_USE_EPOLL` - If set to true, Netty uses epoll instead of NIO.
 * `CASUAL_INBOUND_USE_EPOLL` - If set to true, Netty uses epoll instead of NIO.
+* `CASUAL_INBOUND_INITIAL_DELAY` - Delay the startup of the inbound server. Default is no delay.
 
 NB - if a `CASUAL_CONFIG_FILE` is provided, it take precedence over the `CASUAL_INBOUND_STARTUP_MODE` setting.
```

Comment:
Add note in the documentation about "when" the delay is implemented in the case of startup mode triggered. I.e. we wait for triggered application to complete and then delay. Otherway around could be confusing as you might think the delay and the wait run concurrently.